server/server/src/security.ts

```diff
@@ -99,12 +99,17 @@ export class SecurityClientStorage implements Storage {
       if (typeof querySpace === 'string') {
         if (!spaces.has(querySpace)) throw new PlatformError(new Status(Severity.ERROR, Code.AccessDenied, {}))
       } else {
-        if (querySpace.$in?.every((space) => spaces.has(space)) === false) {
+        if ((querySpace.$in?.every((space) => spaces.has(space))) === false) {
           throw new PlatformError(new Status(Severity.ERROR, Code.AccessDenied, {}))
         }
+        if (querySpace.$like !== undefined) {
+          const like = querySpace.$like
+          const querySpaces = querySpace.$in ?? [...spaces.values()]
+          query.space = { $in: querySpaces.filter(p => p.match(like)) }
```

Comment:
This will not work, since like is with our format, and match accept regular expression.